src/table/tableview/examples/table-view-basic.js

```diff
@@ -20,6 +20,7 @@
   * <ul style='list-style-type: none'>
   *   <li>.header     - (string) Text label for a column header
   *   <li>.itemField    - (string) Item field to associate with a particular column.
+  *   <li>.templateFn - (function) (optional) Template function used to render each cell of the column. Example: <pre>templateFn: value => `<span class="text-danger">${value}</span>`</pre>
```

Comment:
Please add some info. on the pros and cons of using templateFn as opposed to htmlTemplate.  Pro: Is more performant than ```htmlTemplate```.  Con: Doesn't support AngularJS directives in the template, therefore it doesn't support things like ng-click.